.gitignore

```diff
@@ -36,4 +40,4 @@ yarn-debug.log*
 /.idea
 
 # Ignore settings
-/config/application.yml
\ No newline at end of file
+/config/application.yml
```

Comment:
This one is not actually needed for us - it's the Settings patter which is the alternative for .env file. IT got here because I showed the example of .gitignore and it was there